src/Application/Cli/PreviewCommand.php

```diff
@@ -77,18 +78,18 @@ protected function execute(InputInterface $input, OutputInterface $output)
             return 1;
         }
 
+        if (!is_executable($input->getOption('livereload'))) {
+            $output->writeln('<error>Impossible to launch Livereload, did you forgot to install it with "npm install -g livereload"?</error>');
+
+            return 1;
+        }
```

Comment:
So livereload will always be launched? Actually what I mean is that we don't want to break BC so people not having livereload installed should not have errors and I wonder what will happen for them